official/resnet/keras/keras_common.py

```diff
@@ -38,6 +38,42 @@
 TRAIN_TOP_1 = 'training_accuracy_top_1'
 
 
+LR_SCHEDULE = [    # (multiplier, epoch to start) tuples
```

Comment:
hm, i think this schedule and this warmup process is only applicable to imagenet.. maybe you can just use it from there?